recursio-client/Global/PostProcess.gd

```diff
@@ -3,53 +3,53 @@ extends Node
 var _post_process_tool_scene = preload("res://addons/PostProcessingTool/PostProcessing_tool.tscn")
 
 var _pp
-var _tween
 
-#glitch
-var glitch :bool = false setget set_glitch, get_glitch
-# vignette
-var vignette :bool = false setget set_vignette, get_vignette
-var vignette_softness: float = 1.0 setget set_vignette_softness, get_vignette_softness
+# Glitch
+var glitch :bool = false setget set_glitch
+
+# Vignette
+var vignette :bool = false setget set_vignette
+var vignette_softness: float = 1.0 setget set_vignette_softness
+
+# Chromatic Aberration
+var chromatic_ab_strength: float = 1.0 setget set_chromatic_ab_strength
+
+signal shaking_camera(amount, speed)
+
 
 
 func _ready():
-	_tween = Tween.new()
-	add_child(_tween)
 	_pp = _post_process_tool_scene.instance()
 	get_parent().call_deferred("add_child", _pp)
 
 
-func remove_animation(property: NodePath):
-	_tween.remove(self, property)
+func animate_property(property: String, initial_val, final_val, duration: float, trans_type = Tween.TRANS_LINEAR, ease_type = Tween.EASE_OUT):
+	var tween = Tween.new()
+	add_child(tween)
+	tween.interpolate_property(self, property, initial_val, final_val, duration, trans_type, ease_type)
+	tween.start()
+	tween.connect("tween_all_completed", tween, "queue_free")
```

Comment:
I'm not sure it matters, but isn't very inefficient to always instance a new tween and then free it on completion?